static/js/main.js

```diff
@@ -125,4 +105,11 @@ window.onload = function() {
         return false;
     });
 
-}
\ No newline at end of file
+});
+
+// document ready handler
+document.onreadystatechange = function () {
+  if (document.readyState === "interactive") {
+    initApplication();
+  }
+}
```

Comment:
This is so the browser knows when to run scripts which do not rely on jQuery.

http://devdocs.io/dom/document/readystate